test/system/post_test.rb

```diff
@@ -119,32 +119,33 @@ def assert_page_does_not_reload(message = "page should not reload")
     # Make sure that image has been uploaded
     page.assert_selector('#preview img', count: 1)
   end
-  
+
   test "changing and reverting versions works correctly for wiki" do
-    wiki = nodes(:wiki_page)
+    visit '/wiki/wiki-page-path/'
```

Comment:
Why did you change this? Also could you please explain how my test is affecting others?

Reply:
I am trying to figure that out. Every time I submit a PR with system tests, your test fails.
Let's wait for Travis, but it might be possible that it's fetching the wrong row, so I'll have to update it again.

Reply:
On localhost, it works fine though.